### ISSUE TITLE
Nesting @supports got wrong references

### ISSUE THREAD

**Issue by sparanoid** [less/less.js#2750]
input.less:

```css
.alert-popup {
  background: fade(#888, 96%);

  @supports (-webkit-backdrop-filter: blur(15px)) {
    background: fade(#888, 60%);
    -webkit-backdrop-filter: blur(15px);
  }
}
```

output.css:

```css
.alert-popup {
  background: fade(#888, 96%);
}

@supports (-webkit-backdrop-filter: blur(15px)) {
  background: fade(#888, 60%);
  -webkit-backdrop-filter: blur(15px);
}
```

expected.css:

```css
.alert-popup {
  background: fade(#888, 96%);
}

@supports (-webkit-backdrop-filter: blur(15px)) {
  .alert-popup {
    background: fade(#888, 60%);
    -webkit-backdrop-filter: blur(15px);
  }
}
```

**Comment by seven-phases-max** [comment]
Proper bubbling of most of nested directives is supported since [`2.5.0`](https://github.com/less/less.js/blob/master/CHANGELOG.md#250).

**Comment by seven-phases-max** [comment]
Closing as implemented in [`2.5.0`](https://github.com/less/less.js/blob/master/CHANGELOG.md#250).
